build-tools-internal/src/main/groovy/elasticsearch.stable-api.gradle

```diff
@@ -45,8 +45,8 @@ BuildParams.bwcVersions.withIndexCompatible({ it.onOrAfter(Version.fromString(ex
   }
 
   jarApiComparisonTask.configure {
-    onlyIf {
-      !Os.isFamily(Os.FAMILY_WINDOWS)
+    onlyIf("OS != windows") {
```

Comment:
@rjernst Is there a reason we can't run these tasks on Windows?

Reply:
I don't think there is a reason they can't run, but @williamrandolph and @pgomulka would know better.